azext_edge/edge/_help.py

```diff
@@ -780,6 +780,701 @@ def load_iotops_help():
         short-summary: Dataflow endpoint management.
     """
 
+    helps[
+        "iot ops dataflow endpoint create"
+    ] = """
+        type: group
+        short-summary: Create or replace a dataflow endpoint resource.
+    """
+
+    helps[
+        "iot ops dataflow endpoint create adls"
+    ] = """
+        type: command
+        short-summary: Create or replace a dataflow endpoint resource for Azure Data Lake Storage Gen2.
+        long-summary: For more information on Azure Data Lake Storage Gen2 dataflow endpoint, see https://aka.ms/adlsv2.
+
+        examples:
+        - name: Create or replace a dataflow endpoint resource with minimum input.
+          text: >
+            az iot ops dataflow endpoint create adls
+            --name myendpoint
+            --instance mycluster-ops-instance
+            --resource-group myresourcegroup
+            --storage-account mystorageaccount
+        - name: Create or replace a dataflow endpoint resource using user assigned managed identity authentication method.
+          text: >
+            az iot ops dataflow endpoint create adls
+            --name myendpoint
+            --instance mycluster-ops-instance
+            --resource-group myresourcegroup
+            --storage-account mystorageaccount
+            --client-id 425cb1e9-1247-4cbc-8cdb-1aac9b429696
+            --tenant-id bca45660-49a2-4bad-862a-0b9459b4b836
+            --scope "https://storage.azure.com/.default"
```

Comment:
Why not have that scope be the default?  https://learn.microsoft.com/en-us/azure/storage/blobs/authorize-access-azure-active-directory?tabs=dotnet#microsoft-authentication-library-msal 

Or are the use cases of different scopes common enough where a default doesn't make sense?

Reply:
another good catch, thanks! 
<img width="289" alt="image" src="https://github.com/user-attachments/assets/204db5cf-9bc6-4092-9c91-8b9fd2c9e8a4" />